### DISCUSSION TITLE
Include others css and js

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5295] [Q&A] [answered]
I have a template with it's css and js files and i put it on public folder and i want to use it into my header for my site, when i include it using asset {{ }} it is unknown. How can i do that ? It is neccessary

**Comment by jasonvarga** [comment, marked as answer]
Just use the URL. No need for Statamic specific stuff.

```html
<script src="/js/script.js"> <!-- public/js/script.js -->
```
